### PR TITLE
chore: add creating api key section [4933]

### PR DESCRIPTION
**Pull request by chocolateofpain**
This PR renames the 'Team' section in the docs to 'Accounts and Users' and adds a 'Creating an API key' section to it.
It updates the URLs in other sections to reflect the switch from `account/whateversection` to `settings/account/somesection`.
 

What is not covered in this PR:
- updating the screenshots used of the account settings in other sections

https://user-images.githubusercontent.com/76956254/141277507-e7d3c4bc-cfc7-4537-bcb1-c65662d11ab0.mov

### CODE REVIEW

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: Creating an API key
+weight: 2
+menu:
+  docs:
+    parent: "Accounts & Users"
+aliases:
+- "/docs/accounts-users/creating-api-key/"
+---
+
+The Checkly Public API uses API keys to authenticate requests. API keys are unique to the user and not tied to accounts.
```
Comment (brampling) [inline site/content/docs/accounts-and-users/creating-api-key.md:11]:
```suggestion
The Checkly public API uses API keys to authenticate requests. API keys are unique to the user and not tied to an account. 
```

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+menu:
+  docs:
+    parent: "Accounts & Users"
+aliases:
+- "/docs/accounts-users/creating-api-key/"
+---
+
+The Checkly Public API uses API keys to authenticate requests. API keys are unique to the user and not tied to accounts. 
+
+Be aware that Account-based API keys will be deprecated soon. Please create an API key for your user account in [User Settings](https://app.checklyhq.com/settings/user/).
```
Comment (brampling) [inline site/content/docs/accounts-and-users/creating-api-key.md:13]:
```suggestion
Be aware that account-based API keys will be deprecated soon. Please create an API key for your user account in [User Settings](https://app.checklyhq.com/settings/user/).
```

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+**Steps to create an API key:** 
+
+1. Go to the API keys section in the [User Settings](https://app.checklyhq.com/settings/user/)
+2. Click **Create new API key**. A modal pops up.
```
Comment (brampling) [inline site/content/docs/accounts-and-users/creating-api-key.md:19]:
```suggestion
2. Click **Create new API key**.
```

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+The Checkly Public API uses API keys to authenticate requests. API keys are unique to the user and not tied to accounts. 
+
+Be aware that Account-based API keys will be deprecated soon. Please create an API key for your user account in [User Settings](https://app.checklyhq.com/settings/user/).
+
+
+**Steps to create an API key:** 
+
+1. Go to the API keys section in the [User Settings](https://app.checklyhq.com/settings/user/)
+2. Click **Create new API key**. A modal pops up. 
+3. Enter a name for the API key.
```
Comment (brampling) [inline site/content/docs/accounts-and-users/creating-api-key.md:20]:
```suggestion
3. Enter a name for the API key in the dialog box.
```

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Use the API key as a Bearer token in the Authorization header when calling the Checkly API, e.g.
+
+```console
+curl -H "Authorization: Bearer my_token" -H "X-Checkly-Account: my_account_ID" https://api.checklyhq.com/v1/checks
+```
+
+## Deleting an API key
+
+Only the user can delete an API key. To delete an API key click on the **Delete** icon in the Api Keys section of the User Settings.
```
Comment (brampling) [inline site/content/docs/accounts-and-users/creating-api-key.md:34]:
```suggestion
Only the user can delete an API key. To delete an API key click on the **Delete** icon in the API Keys section of the [User Settings](https://app.checklyhq.com/settings/user/). 
```

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+```console
+curl -H "Authorization: Bearer my_token" -H "X-Checkly-Account: my_account_ID" https://api.checklyhq.com/v1/checks
+```
+
+## Deleting an API key
+
+Only the user can delete an API key. To delete an API key click on the **Delete** icon in the Api Keys section of the User Settings. 
+
+If you wish to revoke access of a user to an account, remove the user from the team in the Account Settings.
```
Comment (brampling) [inline site/content/docs/accounts-and-users/creating-api-key.md:36]:
```suggestion
If you wish to revoke access of a user to an account, remove the user from the team in the [Team section of Account Settings](https://app.checklyhq.com/settings/account/team). 
```

**Review by brampling — CHANGES_REQUESTED** [review]

File: site/content/docs/accounts-and-users/creating-api-key.md (hunk 2 above)
Comment (ragog) [reply site/content/docs/accounts-and-users/creating-api-key.md:13]: @chocolateofpain let's call this out with a ">"

File: site/content/docs/accounts-and-users/creating-api-key.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+2. Click **Create new API key**. A modal pops up. 
+3. Enter a name for the API key.
+4. Click **Create new API key**
+5. The new API key is shown and can be copied. Make sure to copy it immediately as it won't be shown again.
+
+## Using an API key
+
+Use the API key as a Bearer token in the Authorization header when calling the Checkly API, e.g.
+
+```console
```
Comment (ragog) [inline site/content/docs/accounts-and-users/creating-api-key.md:28]: @chocolateofpain I don't see any highlights. Let's try with ```sh

**Review by ragog — CHANGES_REQUESTED** [review]
It's going in the right direction but needs a bit of polish.

**Review by brampling — APPROVED** [review]

**Review by ragog — APPROVED** [review]

**Pull request merged**